powersimdata/data_access/data_access.py

```diff
@@ -442,9 +294,9 @@ def push(self, file_name, checksum, change_name_to=None):
         self.move_to(file_name, change_name_to=backup)
 
         values = {
-            "original": self.join(self.root, new_name),
-            "updated": self.join(self.root, backup),
-            "lockfile": self.join(self.root, "scenario.lockfile"),
+            "original": posixpath.join(self.root, new_name),
+            "updated": posixpath.join(self.root, backup),
+            "lockfile": posixpath.join(self.root, "scenario.lockfile"),
```

Comment:
This would be something that needs to be changed via `os` so that our client and server setup will support non-linux servers?

Reply:
Yeah there are several things we'd need to change in that case. As far as I know, non linux servers aren't anywhere on our roadmap.